conseil-lorre/src/main/scala/tech/cryptonomic/conseil/indexer/tezos/TezosFeeOperations.scala

```diff
@@ -26,17 +26,21 @@ private[tezos] object TezosFeeOperations extends LazyLogging {
     "endorsement"
   )
 
-  /**
-    * Calculates average fees for each operation kind and stores them into a fees table.
+  /** Calculates average fees for each operation kind and stores them into a fees table.
+    * The computation will use a limited number of fees, as the result of a selection window in days.
+    * Only blocks belonging within such window in the past, relative to the calculation moment, will be considered.
+    *
     * @param numberOfFeesAveraged a limit on how many of the latest fee values will be used for averaging
-    * @param ex the needed ExecutionContext to combine multiple database operations
+    * @param selectionWindow the max number of days back from the current block timestamp to use when averaging
```

Comment:
The parameter type seems to take an arbitrary duration but you seem to indicate here that it's always in days.

Reply:
As explained below, the input is then approximated up to the day